src/main/java/Chitti.java

```diff
@@ -0,0 +1,205 @@
+                        System.out.println("\t" + list.get(taskIndex).toString());
+                    }
+
+                } else if (input.startsWith("unmark ")) {
+                    if (list.isEmpty()) {
+                        throw new ChittiException("You have no tasks to unmark! Add some tasks first."); // when trying to unmark an empty list
+                    }
+
+                    String[] parts = input.split(" ");
+                    if (parts.length < 2) {
+                        throw new ChittiException("Please specify a task number. Use the following format: unmark <number>"); // Using the 'unmark' function without specifying the task number
+                    }
+
+                    int taskIndex = Integer.parseInt(parts[1]) - 1;
+                    if (taskIndex < 0 || taskIndex >= list.size()) {
+                        throw new ChittiException("Task " + (taskIndex + 1) + " doesn't exist! You have " + list.size() + " tasks."); // when trying to unmark a task number that is outside the list range
+                    }
+
+                    if (!list.get(taskIndex).isMarked()) {
+                        System.out.println("Task " + (taskIndex + 1) + " is already unmarked.");
+                    } else {
+                        list.get(taskIndex).markAsNotDone();
+                        System.out.println("Awwww, I've marked this task as not done yet:");
+                        System.out.println("\t" + list.get(taskIndex));
+                    }
+
+                } else if (input.equals("todo")) {
+                    throw new ChittiException("The description of a todo cannot be empty. Use the following format: todo <description>"); // when trying to use 'todo' without any task description
+
+                } else if (input.startsWith("todo ")) {
+                    String description = input.substring(5).trim();
+                    if (description.isEmpty()) {
+                        throw new Exception("The description of a todo cannot be empty. Use the following format: todo <description>"); // when trying to use 'todo' with a space but without any task description
+                    }
+
+                    ToDo newToDo = new ToDo(description);
+                    list.add(newToDo);
+
+                    System.out.println("Got it! I've added this task:");
+                    System.out.println("\t" + newToDo.toString());
+                    System.out.println("Now you have " + list.size() + " task(s) in the list");
+
+                } else if (input.equals("deadline")) {
+                    throw new ChittiException("The description of a deadline cannot be empty. Use the following format: deadline <description> /by <duedate>"); // when trying to use 'deadline' without any task description
+
+                } else if (input.startsWith("deadline ")) {
+                    String remainingInput = input.substring(9).trim();
```

Comment:
would be better if you do split by whitespace instead of substring at index 9 instead. this can prevent miscalculation in character index or arrayindexoutofbounds exception.
```suggestion
                    String remainingInput = input.split(" ", 2)[1].trim();
```